config/settings.py

```diff
@@ -3,7 +3,7 @@
 
 class Settings(BaseSettings):
     database_url: str = "postgresql+asyncpg://authdb_owner:V97gEkaPchLf@ep-plain-water-a5scbycn.us-east-2.aws.neon.tech/authdb"
-    secret_key: str = "e705c2325058744e7328d842be8da79a1f7c934363b1f061c7934d4804f3d360"
+    secret_key: str = "empty_string_1"
     algorithm: str = "HS256"
```

Comment:
Hardcoding sensitive information, such as database URLs and secret keys, in source code is a security risk. This practice exposes credentials and secrets to anyone who has access to the source code, increasing the risk of unauthorized access. It's recommended to use environment variables or secure vaults for storing sensitive information.

**Label**: security

**Code suggestion**

```python
database_url: str = os.getenv("DATABASE_URL", "default_database_url")
secret_key: str = os.getenv("SECRET_KEY", "default_secret_key")
```